SUI/SUI/ContentView.swift

```diff
@@ -7,18 +7,159 @@
 
 import SwiftUI
 
+/// Экран с машинами
 struct ContentView: View {
-    var body: some View {
+    // MARK: - Private enum
+    private enum Constants {
+        static let wheelsType = ["колпаки", "литые", "деревянные"]
+        static let carModel = "Марка машины:"
+        static let carCost = "Стоимость машины:"
+        static let carMaxSpeed = "Максимальная скорость:"
+        static let wheelsTypeText = "Вид дисков:"
+        static let takeCreditText = "Взять кредит на сумму:"
+        static let chooseWheelsType = "Выберите вид дисков"
+        static let emptyString = ""
+        static let buyCarButtonLabel = "Купить машину"
+        static let buyCarAlertText = "Купить машину"
+        static let yesButtonText = "Да"
+        static let randomCarButtonLabel = "Выбрать машину наугад"
+        static let luckyButtonText = "Мне повезет"
+        static let shareImageName = "square.and.arrow.up"
+        static let vStackSpacing = 15
+        static let carImageWidth = 350
+        static let carImageHeight = 250
+        static let shareButtonHeight = 30
+        static let shareButtonWidth = 30
+        
+    }
+    
+    // MARK: - Body
+
+     var body: some View {
+        VStack(spacing: CGFloat(Constants.vStackSpacing)) {
+            Form {
+                Section {
+                    setupCarImage
+                }
+                Section {
+                    setupCarPicker
+                    Text("\(Constants.carModel) \(cars[segmentIndex].carName)")
+                    Text("\(Constants.carCost) \(cars[segmentIndex].carCost)")
+                    Text("\(Constants.carMaxSpeed) \(cars[segmentIndex].carMaxSpeed)")
+                    setupWheelsTypePicker
+                }
+                Section {
+                    setupCreditSlider
+                    setupTransferCashAlert
+                    setupRandomCarActionSheet
+                }
+            }
+            setupShareShown
+        }
+    }
+    
+    
+    // MARK: - State private properties
+    
+    @State private var segmentIndex = 0
```

Comment:
```suggestion
    @State private var segmentIndex = 0
     @State private var wheelsIndex = 0
     @State private var sliderValue = 0.0
     @State private var isBuyingButtonShown = false
     @State private var isRandomCarActionSheetShown = false
     @State private var isShareSheetShown = false
```
Создай вью модель и вынеси ее